### DISCUSSION TITLE
Handle GPS Distruptions

### DISCUSSION THREAD

**Discussion by YoavOb** [fleaflet/flutter_map#1843] [Q&A]
Hello,
My app is experiencing some GPS distruptions. I need to intervene in the re-centering of the map, and avoid re-centering if the provided location doesn't make sense (out of country bounds).
What is the best practice to solving this issue ?

**Comment by josxha** [comment]
flutter_map has no build-in support for GPS. Are you using some other plugin? (For example flutter_map_location_marker?)

**Comment by ReinisSprogis** [comment]
It makes sense to follow marker even if accuracy is significantly decreased. At least user knows his device GPS accuracy is not good. However you can measure distance from previous location to current location and if it is bigger than X then don't update location markers position.
